### PR TITLE
ytmdl: 2021.06.26 -> 2021.08.01

### PR DESCRIPTION
**Pull request by marsam**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 136082` at da21ff7a run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built successfully:</summary>
  <ul>
    <li>python38Packages.youtube-search-python</li>
    <li>python39Packages.youtube-search-python</li>
    <li>ytmdl</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-python-tests.md">no-python-tests</a>

Test runner could not discover any test cases: ‘Ran 0 tests in 0.000s’
Near pkgs/development/python-modules/youtube-search-python/default.nix:19:0:
```
   |
19 |     description = "Search for YouTube videos, channels & playlists & get video information using link WITHOUT YouTube Data API v3";
   | ^
```
</li>
  </ul>
</details>

File: pkgs/development/python-modules/youtube-search-python/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, isPy27, fetchPypi, httpx }:
+
+buildPythonPackage rec {
+  pname = "youtube-search-python";
+  version = "1.4.7";
+
+  disabled = isPy27;
```
Comment (fabaff) [inline pkgs/development/python-modules/youtube-search-python/default.nix:7]:
```suggestion
  disabled = pythonOlder "3.6";;
```

see https://github.com/alexmercerind/youtube-search-python/blob/main/setup.py#L27. Just to be on the safe site.

File: pkgs/development/python-modules/youtube-search-python/default.nix [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "7f93d9ecfd9b965dc93782d8174b1c1888f8900e2a303254037ba34e1d0ebed4";
+  };
+
+  propagatedBuildInputs = [ httpx ];
+
```
Comment (fabaff) [inline pkgs/development/python-modules/youtube-search-python/default.nix:15]: Adding `doCheck = false;` would make `nixpkgs-hammering` happier.

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 136082` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>python38Packages.youtube-search-python</li>
    <li>python39Packages.youtube-search-python</li>
    <li>ytmdl</li>
  </ul>
</details>

**Pull request merged**